doc/sphinx/source/quickstart/installation.rst

```diff
@@ -500,6 +510,14 @@ and run
 This command installs many of the required dependencies from conda, including
 the ESMValCore package and Python, R, and NCL interpreters.
 
+**MacOSX note:** ESMValTool functionalities in Julia, NCL, and R are not
```

Comment:
I wonder if this is really true. Did you try
- [installing Julia](https://julialang.org/downloads/platform/#macos), running `esmvaltool install Julia` and then `esmvaltool run examples/recipe_julia.yml`
- [installing R (without using of conda)](https://cran.r-project.org/bin/macosx/), running `esmvaltool install R` and then `esmvaltool run examples/recipe_r.yml`

Reply:
Thanks for those suggestions, @bouweandela ! I will flag this and address that at a later point. Currently it is beyond the scope of this PR, but it would be good to assess in what way we can make ESMValTool compatible with OSX fully, with multi-language support.